src/PageView.js

```diff
@@ -0,0 +1,11 @@
+import View from './View';
+
+export default class PageView extends View {
+	/**
+	 * Special type of mark that is going to be looked up in
+	 * performance.navigation.timing.navigationStart instead of PerformanceTimeline
+	 *
+	 * See: https://github.com/w3c/user-timing/issues/22
+	 */
+	startMarkName = 'navigationStart';
```

Comment:
sort of irrelevant if we simplify, but in cases like this where a subclass just overrides a single property, consider leveraging the `viewConfig` constructor arg to pass in a particular property, e.g.

```js
new View({ startMarkName: 'navigationStart', ... }); // roughly equivalient to `new PageView({ ... })`
```